docs/backup.md

```diff
@@ -38,6 +38,20 @@ Now you can move that file to your backup location.
 
 To restore the backup, just copy the backup file to the data folder of your new installation and rename it to `db.sqlite3`. Then start the Docker container.
 
+#### Custom `user`
```

Comment:
I feel "custom user" might be a bit confusing, as there is no such option documented in linkding, and it actually refers to a docker option. 

Personally I'm a bit confused about the new commands as well - it only changes the folder, however in the issue you mentioned that you used a command that also sets a user for the docker command:
```
$  docker exec -u "${PUID}:${PGID}" -it linkding python manage.py backup /etc/linkding/data/backup.sqlite3
```

Instead of documenting this twice, maybe it would make more sense to:
- Create a `backups` folder in the bootstrap script of the docker container, set appropriate permissions on that, and then just reference that folder in the existing documentation
- Or change existing documentation to just use `/etc/linkding/data`

Reply:
> Or change existing documentation to just use /etc/linkding/data

I got concepts mixed up. I do agree that changing the documentation is sufficient. Will address that.